src/po/Make_mvc.mak

```
@@ -34,13 +68,443 @@ LSFLAGS = /b /on /l /s
 INSTALLDIR = $(VIMRUNTIME)\lang\$(LANGUAGE)\LC_MESSAGES
 
 .SUFFIXES:
-.SUFFIXES: .po .mo .pot
+.SUFFIXES: .po .mo .pot .ck
+
+all : $(MOFILES) $(MOCONVERTED)
+
+# 27.10.23, added by Restorer
+.po.ck :
+	$(VIMEXE) -u NONE -e -X -S check.vim -c "if error == 0 | q | endif" -c cq $<
+	$(TOUCH)
+
+# 27.10.23, added by Restorer
+check : $(CHECKFILES)
+
+# 27.10.23, added by Restorer
+checkclean :
+	$(RM) *.ck
+
+# 27.10.23, added by Restorer
+converted : $(MOCONVERTED)
+
+
+# 27.10.23, added by Restorer
+nl.po :
+	@( echo \# > nl.po )
+
+# 27.10.23, added by Restorer
+# Norwegian/Bokmal: "nb" is an alias for "no".
+nb.po : no.po
+	$(CP) no.po nb.po
+
+# 27.10.23, added by Restorer
+# Convert ja.po to create ja.sjis.po.
+ja.sjis.po : ja.po
+	@$(MAKE) -nologo -f Make_mvc.mak sjiscorr
+	-$(RM) $@
+!IF EXIST ("$(GETTEXT_PATH)\msgconv.exe")
+	"$(GETTEXT_PATH)\msgconv.exe" -t CP932 $? | .\sjiscorr.exe > $@
+!ELSEIF DEFINED (ICONV)
+	$(ICONV) -f UTF-8 -t CP932 $? | .\sjiscorr.exe > $@
+!ELSE
+	powershell -nologo -noprofile -Command\
+	[System.IO.File]::WriteAllText(\"$@\",\
+	[System.IO.File]::ReadAllText(\"$?\",\
+	[System.Text.Encoding]::GetEncoding(65001)),\
+	[System.Text.Encoding]::GetEncoding(932))
```

How about adding indents and spaces for readability?
```suggestion
	powershell -nologo -noprofile -Command \
		[System.IO.File]::WriteAllText(\"$@\", \
		[System.IO.File]::ReadAllText(\"$?\", \
		[System.Text.Encoding]::GetEncoding(65001)), \
		[System.Text.Encoding]::GetEncoding(932))
```
(Same for other PowerShell commands.)

---

Did it. Thank you.